packages/tools/pie-icons-webc/README.md

```diff
@@ -0,0 +1,113 @@
+
+# pie-icons-webc
+
+Shared PIE Icon Components for Web Components.
+
+This package generates an Web Component iconset for applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into single file components that can be imported into applications.
+
+---
+
+[![npm version](https://img.shields.io/npm/v/@justeattakeaway/pie-icons-webc.svg)](https://img.shields.io/npm/v/@justeattakeaway/pie-icons-webc.svg)
+
+---
+## Usage
+
+### Installation
+
+Add the module to your project
+
+```bash
+yarn add @justeattakeaway/pie-icons-webc
+```
+
+### Applications
+
+```js
+// Only import what you need!
+import { IconAppRestaurant, IconAppRestaurantLarge } from '@justeattakeaway/pie-icons-webc';
+```
+
+Within the context of a Vue app, that will look like:
+
+  ```js
+  <template>
+      <icon-calendar />
+      <icon-app-restaurant size="xs"/>
+  </template>
+
+  <script>
+  import { IconAlcoholFilled } from '@justeattakeaway/pie-icons-webc';
+  </script>
+  ```
+
+Within the context of a React app, that will look like:
+
+  ```js
+import { IconAppRestaurant } from '@justeattakeaway/pie-icons-webc/icons/IconAppRestaurant';
+import * as React from 'react';
+import { createComponent } from '@lit-labs/react';
+
+export const IconAlcohol = createComponent({
+    displayName: 'IconAlcohol',
+    elementClass: IconAppRestaurant,
+    react: React,
+    tagName: 'icon-app-restaurant',
+    events: { },
+});
+
+  <template>
+      <IconAlcohol size="l"/>
```

Comment:
I think this should be IconAppRestaurant

Reply:
ah so should be IconAlcohol because I'm referencing the wrapper on line 50 in the template 😄